app/containers/Applications/ApplicationReviewValidationContainer.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  faCaretDown,
+  faCaretUp,
+  faCheck,
+  faExclamation,
+  faExclamationTriangle,
+  faTimes
+} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import React, {useState} from 'react';
+import {Col, ListGroup, Row} from 'react-bootstrap';
+import {createFragmentContainer, graphql} from 'react-relay';
+import {ApplicationReviewValidationContainer_applicationRevision} from '__generated__/ApplicationReviewValidationContainer_applicationRevision.graphql';
+
+interface Props {
+  applicationRevision: ApplicationReviewValidationContainer_applicationRevision;
+}
+
+export const ApplicationReviewValidation: React.FunctionComponent<Props> = ({
+  applicationRevision
+}) => {
+  const [expanded, setExpanded] = useState(false);
+
+  const failedValidationItems = applicationRevision.validation.edges.filter(
+    ({node}) => !node.isOk
+  );
+  const passedValidationItems = applicationRevision.validation.edges.filter(
+    ({node}) => node.isOk
+  );
+
+  const failedValidationCount = failedValidationItems.length;
+  const totalValidationCount = applicationRevision.validation.edges.length;
+
+  const validationVariant =
+    failedValidationItems.length > 0 ? 'warning' : 'info';
+
+  const listValidationsWithIcon = (validations, icon) => {
+    if (validations.length === 0) return null;
+
+    return (
+      <ul>
+        {validations.map(({node}) => (
+          <li key={node.validationDescription}>
+            <FontAwesomeIcon icon={icon} fixedWidth className="mr-1" />
+            {node.validationDescription}
+          </li>
+        ))}
+      </ul>
+    );
+  };
+
+  return (
+    <Row className="mt-3">
+      <Col>
+        <ListGroup>
+          <ListGroup.Item variant={validationVariant}>
+            <Row>
+              <Col>
+                <FontAwesomeIcon icon={faExclamationTriangle} fixedWidth />
+                &nbsp;
+                {failedValidationCount === totalValidationCount
```

Comment:
This comparison is only ever true if all the validation checks fail, instead of pass.